### DISCUSSION TITLE
Can I boot up FreeDos?

### DISCUSSION THREAD

**Discussion by hasii2011** [utmapp/UTM#4209] [UTM Help (macOS)]
If I run the .img or .iso files; I get an error message that flashes by too fast for me to see.  I am running on Monterey 12.4

[FreeDos](http://wiki.freedos.org/wiki/index.php/Main_Page)

**Comment by vt887** [comment]
I have boot it sucessfully. MacOS Ventrura 13.6, UTM 4.3.5 (87)
